fractions/logstash/src/main/java/org/wildfly/swarm/logstash/runtime/LogstashCustomizer.java

```diff
@@ -54,11 +57,19 @@ public void customize() {
                     .namedFormatter("logstash")
                     .properties(handlerProps);
 
+            List<String> rootLoggerHandlers = new ArrayList<>();
+
+            RootLogger existingRootLogger = this.logging.subresources().rootLogger();
+            if (existingRootLogger != null && !existingRootLogger.handlers().isEmpty()) {
+                rootLoggerHandlers.addAll(existingRootLogger.handlers());
+            }
+            rootLoggerHandlers.add(logstashHandler.getKey());
+
             this.logging
                     .customFormatter("logstash", "org.jboss.logmanager.ext", "org.jboss.logmanager.ext.formatters.LogstashFormatter",
                                      this.logstash.formatterProperties())
                     .customHandler(logstashHandler)
-                    .rootLogger(this.logstash.level(), logstashHandler.getKey());
+                    .rootLogger(this.logstash.level(), rootLoggerHandlers.toArray(new String[rootLoggerHandlers.size()]));
```

Comment:
Ah, I should have mentioned Level as well.

Anyway, regarding whether Logstash should have its own level and root logger, I think Level is yes, but root logger is no.

## Level
Handler can have its own level, so we should set `swarm.logstash.level` in the definition of logstashHandler like:

``` java
final CustomHandler<?> logstashHandler = new CustomHandler<>("logstash-handler")
    [...]
    .level(logstash.level()); // added
```

## root logger

At least about the level, `swarm.logging.root-logger.level`  or LoggingFraction root-logger level (set by applyDefaults) should be honored rather than Logstash Fraction Level.

``` java
this.logging
   [...]
   .rootLogger(existingRootLogger.level(), rootLoggerHandlers.toArray(new String[rootLoggerHandlers.size()]));
```